Extractor/CMakeLists.txt

```diff
@@ -21,6 +23,8 @@ add_executable(Extractor
     
 	    Zero2/Zero2Extractor.cpp
 	    Zero2/Zero2Extractor.h
-        Zero2/Zero2_DirectoryTable.hpp
-        Zero2/Zero2_FileTable.hpp
-        )
\ No newline at end of file
+        )
+
+add_custom_target(copy-runtime-files ALL
+    COMMAND ${CMAKE_COMMAND} -E copy_directory ${CMAKE_SOURCE_DIR}/resource ${CMAKE_BINARY_DIR}/Obscura
```

Comment:
Change file to the top level's `CmakeLists.txt` and change the line to `configure_file(./resource/ZeroFileDictionary.json ${CMAKE_BINARY_DIR}/Obscura COPYONLY)`